redaxo/src/addons/media_manager/lib/effects/effect_convert2img.php

```diff
@@ -1,7 +1,7 @@
 <?php
 
 /**
- * Benutzt den Konsolen convert Befehl.
+ * Benutzt den Konsolen convert oder ffmpeg Befehl.
```

Comment:
ggf. würde es sinn machen einen separaten effekt dafür zu haben, da der existierende bereits sehr umfangreich ist.

hier interessiert mich aber auch gregors meinung dazu

Reply:
Da er technisch anders arbeitet, würde es separat ganz gut passen aus technischer Perspektive.

Aber andererseits ist es ja der allgemeine convert2img-Effekt, der eigentlich durchaus versuchen soll, aus möglichst vielen Dateiformaten ein sinnvolles Bild zu generieren.
Daher würde ich es denke ich doch hier integrieren in den Effekt.